server/documents/modules/nag.html.eco

```diff
@@ -7,72 +7,502 @@ standalone  : true
+        <div class="ui brown nag"><div class="title">Brown</div><i class="close icon"></i></div> <p></p>
+        <div class="ui black nag"><div class="title">Black</div><i class="close icon"></i></div>
+    </div>
+    <div class="example">
+        <h4 class="ui header">Inverted</h4>
+        <p>A nag can be formatted to appear inverted</p>
+        <div class="ui inverted nag">
+            <div class="title">Inverted</div>
+            <div> This nag has inverted colors</div>
+            <i class="close icon"></i>
+        </div>
+    </div>
+    <div class="example">
+        <h4 class="ui header">Size</h4>
+        <p>A nag can have different sizes</p>
+        <div class="ui mini nag">
+          <div class="title">Mini nag</div>
+          <i class="close icon"></i>
+        </div>
+        <p></p>
+        <div class="ui tiny nag">
+          <div class="title">Tiny nag</div>
+          <i class="close icon"></i>
+        </div>
+        <p></p>
+        <div class="ui small nag">
+          <div class="title">Small nag</div>
+          <i class="close icon"></i>
+        </div>
+        <p></p>
+        <div class="ui large nag">
+          <div class="title">Large nag</div>
+          <i class="close icon"></i>
+        </div>
+        <p></p>
+        <div class="ui big nag">
+          <div class="title">Big nag</div>
+          <i class="close icon"></i>
+        </div>
+        <p></p>
+        <div class="ui huge nag">
+          <div class="title">Huge nag</div>
+          <i class="close icon"></i>
+        </div>
+        <p></p>
+        <div class="ui massive nag">
+          <div class="title">Masssive nag</div>
+          <i class="close icon"></i>
+        </div>
 
     </div>
+  </div>
+  <div class="ui tab" data-tab="examples">
+      <h2 class="ui dividing header">Examples</h2>
+
+      <div class="ui code" data-type="html" data-preview="true">
+        <div class="ui cookie nag">
+          <span class="title">
+            We use cookies to ensure you get the best experience on our website
+          </span>
+          <i class="close icon"></i>
+        </div>
+      </div>
+      <div class="evaluated code" data-type="javascript">
+      // Automatically shows on init if cookie isn't set
+      $('.cookie.nag')
+        .nag({
+          key      : 'accepts-cookies',
+          value    : true
+        })
+      ;
+      </div>
+      <div class="code" data-type="javascript" data-demo="true">
+      // Wont re-appear unless cleared
+      $('.cookie.nag')
+        .nag('show')
+      ;
+      </div>
+      <div class="code" data-type="javascript" data-demo="true">
+      // Clears cookie so nag shows again
+      $('.cookie.nag')
+        .nag('clear')
+      ;
+      </div>
+
 
   </div>
 
+  <div class="ui tab" data-tab="usage">
+      <h2 class="ui dividing header">Usage</h2>
+
+      <div class="no example">
+        <h4 class="ui header"></h4>
```

Comment:
This empty header creates a lot of gap between the above header "Usage" and the example.

Reply:
Fixed by 9d393cdf3a133a6c6f9f31317ce64454de6e4955
Please review again